util/cli/log.go

```diff
@@ -3,24 +3,15 @@ package cli
 import (
 	"encoding/json"
 
-	"gopkg.in/src-d/go-log.v1"
+	gocli "gopkg.in/src-d/go-cli.v0"
+	log "gopkg.in/src-d/go-log.v1"
 )
 
-// LogOptions defines logging flags. It is meant to be embedded in a
-// command struct.
-type LogOptions struct {
-	LogLevel       string `long:"log-level" env:"LOG_LEVEL" default:"info" description:"Logging level (info, debug, warning or error)"`
-	LogFormat      string `long:"log-format" env:"LOG_FORMAT" description:"log format (text or json), defaults to text on a terminal and json otherwise"`
-	LogFields      string `long:"log-fields" env:"LOG_FIELDS" description:"default fields for the logger, specified in json"`
-	LogForceFormat bool   `long:"log-force-format" env:"LOG_FORCE_FORMAT" description:"ignore if it is running on a terminal or not"`
-}
-
-var _ initializer = &LogOptions{}
-
-// Init initializes the default logger factory.
-func (c *LogOptions) init(app *App) {
+// InitLog is similar to go-cli LogOptions.Init, but adds the application name
```

Comment:
it also applies the configuration to logrus.

Reply:
Done in f800074